components/Activity/ActivityCard.tsx

```diff
@@ -63,10 +64,11 @@ export function ActivityCard({
         </Card.Text>
       </Card.Body>
       <Card.Footer>
-        <Row as="small" className="text-muted g-4" xs={1} sm={2}>
+        <Row as="small" className="text-muted g-4" xs={1} sm={1}>
```

Comment:
```suggestion
        <Row as="small" className="text-muted g-4" xs={1}>
```
**Bootstrap 栅格布局**是**移动优先**的，大屏默认使用更小屏的设置。